### PR TITLE
add: add list as default option, remove amsdos unsupported char in as…

### PR DESCRIPTION
**Pull request by jeromelesaux**
I add default option if no option is set and remove unsupported char by amsdos is ascii file.

### CODE REVIEW

**Comment by rgiot** [comment]
It seems I need to fix my test suite after this patch. I'll try to do that during the day and merge this request

File: src/Main.cpp [hunk 1]
```diff
@@ -90,6 +91,7 @@ int main(int argc, char **argv)
 
 	if (ModeListBasic || ModeListHex || ModeListDams || ModeDisaFile || ModeListAscii)
 	{
+		NoOptionSet = true;
```
Comment (rgiot) [inline src/Main.cpp:94]: Are you sur you do not want to put NoOptionset to false ? By putting it to true, the catalog is displayed even when a file is listed.

Personally I would put false.

**Review by rgiot — COMMENTED** [review]
I have a doubt at line 94. Just need a double check before I merge

**Comment by jeromelesaux** [comment]
Yep I prefer to display the catalog if no option is set

**Comment by jeromelesaux** [comment]
But it's up to you

**Comment by rgiot** [comment]
OK. but an option is set if the following test is true  `ModeListBasic || ModeListHex || ModeListDams || ModeDisaFile || ModeListAscii` ;) Have i have misunderstood something ?

**Comment by jeromelesaux** [comment]
Nope you're true. It might be false.

**Pull request merged**